frontend/packages/console-shared/src/utils/icon-utils.ts

```diff
@@ -0,0 +1,9 @@
+import * as _ from 'lodash';
+import * as operatorLogo from '../images/operator.svg';
+
+export const getImageForCSVIcon = (csv) =>
+  _.get(csv, 'spec.icon')
+    ? `data:${_.get(csv, 'spec.icon', [])[0].mediatype};base64,${
```

Comment:
assigning default as `[]` so in case of default it may break as `[0]` would be `undefined` and we would access `mediatype` on that

Reply:
Done.